portal-impl/src/portal.properties

```diff
@@ -2810,8 +2810,8 @@
 
     #
     # Set this to true to use the full hostname for cookie domain values. By
-    # default the domain is trimmed to use only the root domain to allow cross
-    # domain logins.
+    # default, the label for the lowest level in the domain is trimmed to allow
+    # cross domain logins.
```

Comment:
The old behavior was inconsistent if the second level domain had more than 2 labels (e.g., liferay.com.cn). So we're changing it to always remove the left-most (i.e., lowest level) label in the domain unless the domain is already the top private domain (i.e., a second level domain). See https://issues.liferay.com/browse/LPS-51404?focusedCommentId=571709&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-571709 for more info